src/components/signup/SignupState.js

```diff
@@ -520,14 +497,12 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
 
     if (nextRoute === undefined) {
       const ok = await waitForRegistrationToFinish()
-
-      //this will cause a re-render and move user to the dashboard route
       if (ok) {
-        store.set('isLoggedIn')(true)
+        setSuccessfull(() => store.set('isLoggedIn')(true))
```

Comment:
can we move isLoggedIn into a Context?

Reply:
Unfortunately no, because AuthContext describes just the state of login/signup 
Its provider is rendered in the SignupRouter only.

to keep user logged state and user profile we need to create anothter context and put its provider on higher level of the app.
this is a huge refactoring, i will open a separate task for this